lib/model/model_test.go

```diff
@@ -1329,22 +1312,22 @@ func TestAutoAcceptNewFolderPremutationsNoPanic(t *testing.T) {
 					m.Stop()
 					testOs.RemoveAll(id)
 					testOs.RemoveAll(label)
+					os.Remove(wcfg.ConfigPath())
```

Comment:
TempOS still needed above?

Reply:
Yes, from my understanding of this test it is, as all loop iterations use the same id and label, thus failed removals will cause unwanted conflicts.